ingest/template/vanilla_spreadsheet_builder.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+"""
+Given a tabs template and list of schema URLs, will output a spreadsheet in Excel format.
+"""
+
+from .spreadsheet_builder import SpreadsheetBuilder
+
+DEFAULT_INGEST_URL = "http://api.ingest.data.humancellatlas.org"
+DEFAULT_SCHEMAS_ENDPOINT = "/schemas/search/latestSchemas"
+
+
+class VanillaSpreadsheetBuilder(SpreadsheetBuilder):
+    def __init__(self, output_file, hide_row=False):
+        super(VanillaSpreadsheetBuilder, self).create_initial_spreadsheet(output_file, hide_row)
+
+    def build(self, template):
```

Comment:
`template` could be more descriptively called `spreadsheet_tabs_template`.

It would be nice to talk here or in the class or module docstring about the format of the spreadsheet tabs template, and what the process of "spreadsheet building" actually entails - or link to another piece of documentation where this is covered.

Reply:
I see that parts of this are covered in the docstring for `generate_spreadsheet()`, would be great to organize this somehow so there are breadcrumbs between the two.

Reply:
I changed `template` to be `spreadsheet_tabs_template` and also filed https://app.zenhub.com/workspaces/dcp-5ac7bcf9465cb172b77760d9/issues/humancellatlas/ingest-client/38 to generally improve documentation. What you mentioned above is one of the tasks.